modules/osdk-installing-cli.adoc

```diff
@@ -107,8 +107,10 @@ To download the key, run the following command, replacing `<key_id>` with the RS
 key string provided in the output of the previous command:
 +
 ----
-$ gpg --recv-key "<key_id>"
+$ gpg [--keyserver keys.gnupg.net] --recv-key "<key_id>" <1>
```

Comment:
One concern, why add the `[`/`]` here?

Reply:
Because it is optional if you already have a keyserver configured. E.g., in my initial testing of these steps, I did not have to pass this option.

Reply:
ok, got it, LGTM, thanks!